code-challenges.test.js

```diff
@@ -54,3 +114,14 @@ const numbersToAdd3 = []
 
 
 // b) Create the function that makes the test pass.
+
+//create a function called accumSum
+//use .reduce method to check each element in the array and return the sum of all elements in the array?
+//
+
+const accumSum = (sum) => {
+    total = sum
+    return sum.reduce(total)
+}
+
+//my over all thoughts on the majority of these questions is I am lost. I seem to either have trouble understanding the question and the way its written as well as not grasping functions entirely. I am taking coding courses outside of the bootcamp on Javascript functions to get a better understanding through codedacemy.
```

Comment:
The reduce method is great for adding everything together, but often can lead to only a single number return and not an array.  Make sure to google the built in methods that you want to try to get an idea on how they operate and what additional information they may need.